packages/kbn-unified-data-table/src/components/data_table.tsx

```diff
@@ -746,27 +751,29 @@ export const UnifiedDataTable = ({
         showColumnTokens,
         headerRowHeightLines,
         customGridColumnsConfiguration,
+        columnHeaders,
       }),
     [
-      columnsMeta,
+      visibleColumns,
       columnsCellActions,
-      customGridColumnsConfiguration,
+      displayedRows.length,
+      settings,
       dataView,
-      dataViewFieldEditor,
       defaultColumns,
-      displayedRows.length,
-      editField,
-      headerRowHeightLines,
-      isPlainRecord,
       isSortEnabled,
-      onFilter,
-      settings,
-      showColumnTokens,
-      toastNotifications,
+      isPlainRecord,
       uiSettings,
+      toastNotifications,
       valueToStringConverter,
+      onFilter,
+      editField,
       visibleCellActions,
-      visibleColumns,
+      columnsMeta,
+      showColumnTokens,
+      headerRowHeightLines,
+      customGridColumnsConfiguration,
+      columnHeaders,
+      dataViewFieldEditor,
```

Comment:
I don't understand why all those fields changed in their order, I think the only change here should have been that it was added the `columnHeaders` variable?

Reply:
good question, the order got changed Automatically when I click on the option to add columnHeaders onto the list on VScode